.github/workflows/deploy-tobira-opencast.yml

```diff
@@ -0,0 +1,75 @@
+name: Deploy Tobira Opencast
+
+on:
+  push:
+    branches:
+      - main
+      # remove before merging
+      - tobira-deployment
+
+jobs:
+  build:
+    name: deploy tobira opencast
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: cache local maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven-
+
+      - name: setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: install dependencies
+        run: |
+          sudo apt update -q
+          sudo apt install -y -q \
+            bzip2 \
+            gzip \
+            tar \
+            unzip
+
+      - name: install ansible
+        run: pip3 install ansible dnspython
+
+      # We want to switch to -Pallinone once it is available
+      # https://github.com/opencast/opencast/pull/2788
+      - name: build opencast
+        run: |
+          mvn clean install -Pdist \
+            --batch-mode \
+            -DskipTests \
```

Comment:
I always use `-DskipTests=true` locally. Are you telling me the `=true` was not necessary all along? :D

Reply:
I would never tell you that. Keep using the `=true` ;-p

Reply:
@LukasKalbertodt I told you that repeatedly already, though. :stuck_out_tongue: #unqualifieddrivebycommentsftw

Reply:
His fault for never listening to you…